docs/source/cdk/cdk-deployment.rst

```diff
@@ -0,0 +1,30 @@
+CDK deployment
+==============
+Deploy
+~~~~~~
+You will need to make a **copy** of app_template_dev.py file with a different name (app__dev.py) and keep a copy of
+it locally so that it will not be committed. In your own copy there are two important configuration
+items which you can alter:
+
+#. AWS_PROFILE ()
+#. Your initials when deploying to an AWS account with multiple users ()
```

Comment:
What does the trailing `()` do? Are you meaning to put ``<profile>``  and ``<initials>`` in as reference here instead.